core/dbt/include/global_project/macros/python_model/python.sql

```diff
@@ -56,8 +56,8 @@ class config:
         pass
 
     @staticmethod
-    def get(key):
-        return config_dict.get(key)
+    def get(key, default=None):
+        return config_dict.get(key, default=None)
```

Comment:
This python looks slightly off, can you do `default=None` in a method invocation? I would think this should be
```suggestion
    def get(key, default=None):
        return config_dict.get(key, default)
```
since `default` already defaults to `None`.